### PR TITLE
prompt user before allowing location access

### PR DESCRIPTION
**Pull request by jpcloureiro**

### CODE REVIEW

**Comment by jpcloureiro** [comment]
depends on #8.

ready to review after rebase

File: android/src/main/java/com/reactnativecommunity/webview/RNCWebChromeClient.java [hunk 1]
```diff
@@ -208,7 +209,18 @@ public void onGeolocationPermissionsShowPrompt(String origin, GeolocationPermiss
...
+            builder.setMessage(alertMessage);
+            builder.setCancelable(false);
+            builder.setPositiveButton("Allow", (dialog, which) -> {
+                callback.invoke(origin, true, false);
+            });
+            builder.setNegativeButton("Don't allow", (dialog, which) -> {
+                callback.invoke(origin, false, false);
+            });
+            AlertDialog alertDialog = builder.create();
+            alertDialog.show();
```
Comment (tommasini) [inline android/src/main/java/com/reactnativecommunity/webview/RNCWebChromeClient.java:223]: We talked about adding the this.delayed function here on the mobile patch, will this be addressed later ?

**Review by tommasini — CHANGES_REQUESTED** [review]

File: android/src/main/java/com/reactnativecommunity/webview/RNCWebChromeClient.java (hunk 1 above)
Comment (tommasini) [reply android/src/main/java/com/reactnativecommunity/webview/RNCWebChromeClient.java:223]: Nevermind it was updated here on this [PR](https://github.com/MetaMask/react-native-webview-mm/pull/10)

**Review by tommasini — APPROVED** [review]
LGTM!

**Review by sethkfman — APPROVED** [review]
LGTM

**Pull request merged**